### ISSUE TITLE
[A11y]The narrator announces same name for all 3 actionable images present under Owner section

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7883]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1067633
Repro Steps:
1. Hit the URL:https://www.nuget.org/
2. Navigate to "Sign in" control and hit enter.
3. Navigate to "Sign in with Microsoft" control and hit enter.
4. Search for a key term (say “json”) from search box.
5. Click on "Newtonsoft.json" control and hit enter.
6. Navigate to "Owners" section present on bottom right of the screen.
7. Turn on the narrator.
8. Check if the narrator announces the correct name for all the 3 images in owner section.

Actual:
The narrator announces the same name like "gravatar" for all the 3 images under owner section.

Same issue is Observed in "NVDA" also.

Expected:
The narrator should announce the names of the 3 images differently as all the 3 images are actionable imags it should have different name.

Notes:
All 3 images in the describes owners section have alt text of 'gravatar'
I am not 100% sure if we can hijack the alt text here, but if we can, we could update the text to read as the owner.

**Comment by ryuyu** [comment]
Looks like we put the alt text. Very easy for us to edit it. Updated to read "username gravatar"
